OSSpecific/src/artofillusion/osspecific/MacOSPlugin.java

```diff
@@ -15,16 +15,21 @@ Changes copyright (C) 2017 by Maksim Khramov
 import artofillusion.ui.*;
 import buoy.event.*;
 import buoy.widget.*;
+import com.apple.eawt.Application;
+import com.apple.eawt.ApplicationEvent;
 import java.awt.*;
 import java.io.*;
-import java.lang.reflect.*;
+import java.util.logging.Level;
```

Comment:
We have not decided on a logging strategy. (#50)

Please refrain from adding logging code that is specific to a certain logger configuration, especially as the core concern here has nothing to do with logging.